src/guide/code-owners.md

```diff
@@ -126,40 +163,37 @@ This list is intended to make it easier to identify which email list to include
 * `java.xml` – Core Libs
 * `java.xml.crypto` – Security
 * `jdk.accessibility` – Client Libs
-* `jdk.aot` – HotSpot Compiler
 * `jdk.attach` – Serviceability
 * `jdk.charsets` – I18n, Core Libs
-* `jdk.compiler` – LangTools
+* `jdk.compiler` – Javac
 * `jdk.crypto.cryptoki` – Security
 * `jdk.crypto.ec` – Security
 * `jdk.crypto.mscapi` – Security
-* `jdk.crypto.ucrypto` – Security
-* `jdk.dynalink` – LangTools
-* `jdk.editpad` – LangTools
+* `jdk.dynalink` – JDK Tools
+* `jdk.editpad` – JShell
 * `jdk.hotspot.agent` – Serviceability
 * `jdk.httpserver` – Net
-* `jdk.incubator.foreign` – LangTools
-* `jdk.incubator.httpclient` – Net
-* `jdk.incubator.vector` – HotSpot Compiler
-* `jdk.internal.ed` – LangTools
+* `jdk.incubator.concurrent` – Core Libs
+* `jdk.incubator.vector` – Compiler
+* `jdk.internal.ed` – JShell
 * `jdk.internal.jvmstat` – Serviceability
-* `jdk.internal.le` – LangTools
-* `jdk.internal.opt` – LangTools
-* `jdk.internal.vm.ci` – HotSpot Compiler
-* `jdk.internal.vm.compiler` – HotSpot Compiler
-* `jdk.internal.vm.compiler.management` – HotSpot Compiler
-* `jdk.jartool` – LangTools
-* `jdk.javadoc` – LangTools
+* `jdk.internal.le` – JShell
+* `jdk.internal.opt` – JDK Tools
+* `jdk.internal.vm.ci` – Compiler
+* `jdk.internal.vm.compiler` – Compiler
+* `jdk.internal.vm.compiler.management` – Compiler
+* `jdk.jartool` – JDK Tools
+* `jdk.javadoc` – Javadoc
 * `jdk.jcmd` – Serviceability
 * `jdk.jconsole` – Serviceability
 * `jdk.jdeps` – Core Libs
 * `jdk.jdi` – Serviceability
 * `jdk.jdwp.agent` – Serviceability
 * `jdk.jfr` – JFR
-* `jdk.jlink` – LangTools
-* `jdk.jpackage` – Client / jpackage
-* `jdk.jshell` – LangTools
-* `jdk.jsobject` – LangTools
+* `jdk.jlink` – JDK Tools
+* `jdk.jpackage` – Client Libs
```

Comment:
"client libs" != "Client" 

Client libs is about the desktop APIs used at run time.
jpackage is an installer technology - for desktops and servers
jdk.jpackage would be better as Core Libs like you have for the jpackage tool, or left as is. Client Libs is definitely wrong.
The existing words try to say "this is the implementation of the jpackage tool, which happens to be owned by the client team but isn't at all a part of client desktop". 
The issue is that this is installer technology and historically all installers were owned by client, since the desktop installers were the main focus.
You should think of this as being a case of wearing two hats.